### PR TITLE
[CDAP-16924] Integration test for pipeline triggers

### PR DESCRIPTION
**Pull request by itsanudeep**
Integration test for triggering a pipeline with a different pipeline.
- Create two pipelines.
- Create inbound trigger for pipeline2 to be triggered when pipeline1 succeeds.
- Create a simple trigger.
- Create a complex trigger with mapping runtime arguments.
- Verifying the trigger is available in pipeline1 as an outbound trigger.

JIRA: https://issues.cask.co/browse/CDAP-16924

### CODE REVIEW

**Comment by ajainarayanan** [comment]
@itsanudeep has 51 failures. Is this PR ready for review?

**Comment by itsanudeep** [comment]
> @itsanudeep has 51 failures. Is this PR ready for review?

@ajainarayanan All the tests were passing locally when I opened the PR, I'll check what's going on.

File: cdap-ui/cypress/integration/pipeline.triggers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        // Delete the pipeline to clean up
+        cy.cleanup_pipelines(headers, TRIGGER_PIPELINE_1);
+        cy.cleanup_pipelines(headers, TRIGGER_PIPELINE_2);
+    });
+
+    it('Should be rendered correctly', () => {
+        cy.visit('/pipelines/ns/default/studio');
+    });
+
+    it('Should create first pipeline and deploy it', () => {
```
Comment (yukiej) [inline cdap-ui/cypress/integration/pipeline.triggers.spec.ts:53]: Could you use the deployAndTestPipeline helper function here? Probably also for the next test block as well.

File: cdap-ui/cypress/integration/pipeline.triggers.spec.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        cy.visit(`/pipelines/ns/default/view/${TRIGGER_PIPELINE_1}`);
+        cy.url().should('include', '/view');
+        cy.get(dataCy('outbound-triggers-toggle')).should('exist');
+        cy.get(dataCy('outbound-triggers-toggle')).click();
+        cy.get(dataCy(`${TRIGGER_PIPELINE_2}-triggered-collapsed`)).should('exist');
+        cy.get(dataCy(`${TRIGGER_PIPELINE_2}-triggered-collapsed`)).click();
+        cy.get(dataCy(`${TRIGGER_PIPELINE_2}-triggered-expanded`)).contains('Succeeds');
+    });
+
+});
```
Comment (yukiej) [inline cdap-ui/cypress/integration/pipeline.triggers.spec.ts:139]: Nit: newline at end of file

**Comment by itsanudeep** [comment]
will open against develop first

**Pull request closed without merging**